requirements.txt

```diff
@@ -5,3 +5,4 @@ cacheout==0.13.1
 better_profanity==0.6.1
 hatesonar
 pymongo
```

Comment:
Should pymongo still be installed by default if we want mongita as the default one?

Reply:
IMO yes, the cost of adding pip dependencies is not really big and allows for easier testing